docs/en/getting_started.md

```diff
@@ -54,24 +71,143 @@ many workers were created when you started faucet. This is because faucet
 automatically detects the number of CPU cores on your computer and creates
 a worker for each core.
 
+To know how many CPU cores you have, you can run the following commands:
```

Comment:
This is amazing, but if you run 

```
faucet start --help
```
 
Faucet should tell you how many CPUs you have on your computer.

```
  -w, --workers <WORKERS>
          The number of threads to use to handle requests
          
          [env: FAUCET_WORKERS=]
          [default: 16]
```

The default tells you what faucet has detected.